api/api.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const EthNamespace = "eth"
-const defaultGasPrice = 8049999872
+const DefaultGasPrice = 8049999872
```

Comment:
probably better to have these default config variables in the config.go

Reply:
Good idea :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/39/commits/7a979d01e592cde5322cc287a71c5e014b2b7494